### ISSUE TITLE
Error: "Exception in ASGI application"

### ISSUE THREAD

**Issue by akarmes** [koxudaxi/local-data-api#60]
**Describe the bug**

Querying local-data-api causes this error:

```
[2020-04-30 14:04:16 +0000] [8] [ERROR] Exception in ASGI application
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/uvicorn/protocols/http/httptools_impl.py", line 385, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/usr/local/lib/python3.7/site-packages/uvicorn/middleware/proxy_headers.py", line 45, in __call__
    return await self.app(scope, receive, send)
  File "/usr/local/lib/python3.7/site-packages/fastapi/applications.py", line 140, in __call__
    await super().__call__(scope, receive, send)
  File "/usr/local/lib/python3.7/site-packages/starlette/applications.py", line 134, in __call__
    await self.error_middleware(scope, receive, send)
  File "/usr/local/lib/python3.7/site-packages/starlette/middleware/errors.py", line 178, in __call__
    raise exc from None
  File "/usr/local/lib/python3.7/site-packages/starlette/middleware/errors.py", line 156, in __call__
    await self.app(scope, receive, _send)
  File "/usr/local/lib/python3.7/site-packages/starlette/exceptions.py", line 73, in __call__
    raise exc from None
  File "/usr/local/lib/python3.7/site-packages/starlette/exceptions.py", line 62, in __call__
    await self.app(scope, receive, sender)
  File "/usr/local/lib/python3.7/site-packages/starlette/routing.py", line 590, in __call__
    await route(scope, receive, send)
  File "/usr/local/lib/python3.7/site-packages/starlette/routing.py", line 208, in __call__
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.7/site-packages/starlette/routing.py", line 41, in app
    response = await func(request)
  File "/usr/local/lib/python3.7/site-packages/fastapi/routing.py", line 129, in app
    raw_response = await run_in_threadpool(dependant.call, **values)
  File "/usr/local/lib/python3.7/site-packages/starlette/concurrency.py", line 25, in run_in_threadpool
    return await loop.run_in_executor(None, func, *args)
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/app/local_data_api/main.py", line 85, in execute_statement
    request.database,
  File "/app/local_data_api/resources/resource.py", line 216, in get_resource
    connected_database = connection.get_dsn_parameters()[
AttributeError: 'Connection' object has no attribute 'get_dsn_parameters'
```

**To Reproduce**

Problem occurs only with MySQLJDBC in Codeship CI. It works great with PostgreSQLJDBC both locally and in Codeship and with MySQLJDBC locally.
 
My Codeship config looks like that:

```yaml
aurora-mysql:
  image: mysql:5.6
  command: --default-authentication-plugin=mysql_native_password
  environment:
    MYSQL_DATABASE: dashboard
    MYSQL_ROOT_PASSWORD: password
aurora-mysql-http:
  image: koxudaxi/local-data-api
  environment:
    ENGINE: MySQLJDBC
    MYSQL_HOST: aurora-mysql
    MYSQL_PORT: 3306
    MYSQL_USER: root
    MYSQL_PASSWORD: password
    RESOURCE_ARN: arn:aws:rds:us-east-1:123456789012:cluster:aurora-mysql
    SECRET_ARN: arn:aws:secretsmanager:us-east-1:123456789012:secret:aurora-mysql
  depends_on:
    - aurora-mysql
 tester:
  image: ...
  environment:
    - AURORA_HOST=aurora-mysql
    - AURORA_ENDPOINT=http://aurora-mysql-http
```

What I can see is that MySQL is operational and unit tests with direct DB connection are green.

**Comment by koxudaxi** [comment]
@akarmes
Thank you for creating this issue.

I have fixed and publish a new image as `0.4.11`

I could not replay the error. But, I guessed a reason for the error. I think I could fix the error.

Could you please test the new version?
Docker daemon may use a cached old version, you should set the new version as a tag of the image.

**Comment by akarmes** [comment]
@koxudaxi New version completely resolves the issue. Thank you so much for such a lightning speed!
